backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/DomainObject/Guards/GuardContentTests.cs

```diff
@@ -336,23 +372,48 @@ private ISchemaEntity CreateSchema(bool isSingleton)
             return Mocks.Schema(appId, NamedId.Of(DomainId.NewGuid(), "my-schema"), new Schema("schema", isSingleton: isSingleton));
         }
 
-        private IContentEntity CreateDraftContent(Status status)
+        private IContentEntity CreateDraftContent(Status status, RefToken? refTokenParam)
         {
             return new ContentEntity
             {
                 Id = DomainId.NewGuid(),
                 NewStatus = status,
-                AppId = appId
+                AppId = appId,
+                CreatedBy = refTokenParam
             };
         }
 
-        private IContentEntity CreateContent(Status status)
+        private IContentEntity CreateContent(Status status, RefToken? refTokenParam = null, ISchemaEntity? schema = null)
         {
             return new ContentEntity
             {
                 Id = DomainId.NewGuid(),
                 Status = status,
-                AppId = appId
+                AppId = appId,
+                CreatedBy = refTokenParam,
+                SchemaId = schema.AppId
```

Comment:
Why poitns schema id to app id?

Reply:
because they have same value and one type - NamedId<DomainId>